CMSgov/design-system
# Wnmgds 1671/mgov focus style

## Summary

- Adds focus color for Medicare.gov
- Updates medicare loki screenshots, looks like there were several that were rendered when something was broken because the new ones look right.

## How to test

`yarn build && yarn build:medicare && yarn start:medicare`, check focus style is now `#C97532`
